Silicon/ArrowlakePkg/Include/Library/PchPcieRpConfig.h

```diff
@@ -0,0 +1,386 @@
+/** @file
+  PCH Pcie root port policy
```

Comment:
header file under include\Library is to define a library class. This header doesn't define it. If it is used only by one module, move to that module. If it is used by multiple modules, move to upper level folder.

Reply:
Moved header file to upper level folder